File: ReservationServiceClient/model/notificationservice/base/NotificationType.cs

public class NotificationType {
    public int id{ get; set; }
    public string notification_type{ get; set; }

    public NotificationType(string notification_type) {
        this.notification_type = notification_type;
    }

    public NotificationType() {
    }

    public NotificationType(int id, string notification_type) {
        this.id = id;
        this.notification_type = notification_type;
    }
}

File: notification-service/src/main/java/rs/raf/notificationservice/domain/NotificationType.java
package rs.raf.notificationservice.domain;

import lombok.*;

import javax.persistence.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
@ToString
@Entity
public class NotificationType {
    @Id
    private Integer id;
    @Column(name = "notification_type")
    private String notification_type;

    public NotificationType(String notification_type) {
        this.notification_type = notification_type;
    }
}
